docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -136,6 +136,120 @@ tasks.named('compileKotlin', org.jetbrains.kotlin.gradle.tasks.KotlinCompilation
 </tab>
 </tabs>
 
+### Migrate from `kotlinOptions{}` to `compilerOptions{}` {collapsible="true"}
+
+Before Kotlin 2.2.0, you could configure compiler options using the `kotlinOptions{}` block. Since the `kotlinOptions{}`
+block is deprecated in Kotlin 2.2.0, this section provides guidance and recommendations for migrating your build
+scripts to use the `compilerOptions{}` block instead.
+
+#### Centralize compiler options and use types
+
+Whenever possible, configure compiler options at the [extension level](#extension-level), and override them for specific tasks
+at [compilation unit level](#compilation-unit-level).
+
+To make your configuration safer and more expressive, avoid using raw strings and prefer strongly typed values. For example, if you have:
```

Comment:
It is not possible to use raw string directly in the new DSL. Maybe better to rephrase here to "avoid using, but migration helpers are available"

Reply:
I didn't know that. In this case, I think we should be direct about it. What do you think?

> You can't use raw strings in the `compilerOptions {}` block, so convert them to typed values.

Reply:
sounds good to me